src/procselfmaps.cpp

```diff
@@ -235,6 +235,7 @@ int ProcSelfMaps::getNextArea(ProcMapsArea* area)
     area -> flags |= MAP_ANONYMOUS;
   }
 
+  area->mmapFileSize = 0;
```

Comment:
Let's initialize `area->mmapFileSize` to `-1`.  The intention of the code is then clearer.  A size of 0 could be understood as somehow mmap'ing a file of file size 0, which is not the intention.